query-planner-js/src/FieldSet.ts

```diff
@@ -21,13 +21,49 @@ export interface Field<
   fieldDef: GraphQLField<any, any>;
 }
 
+/**
+ * Provides a string representation of a field suitable for debugging.
+ *
+ * The format looks like '(a: Int)<A [A1, A2]>' where 'a' is the field name, 'Int' is its type, and '<...>' is its
+ * scope (@see debugScope for details).
+ *
+ * @param field - the field object to convert.
+ * @return a string representation of the field.
+ */
+export function debugField(field: Field) : string {
```

Comment:
I'm wondering if it would make sense to rename this to `debugPrintField` (and similarly for the other formatting functions). On the one hand, that could be confusing because generating a string isn't necessarily what people understand by printing. On the other hand, there is some precedent here, with our own `printFields` and with functions like `printType` and `printSchema` in `graphql-js` for example.

Reply:
In principle, not sure I'm sold on using `print` for something that produces a string for the reason you mention. That said, since this seems to be the convention used by `graphql-js` and some existing federation code, no reason not to improve consistency. Renamed those methods and the "fn" arguments mentioned in your other comments.